src/error.c

```diff
@@ -302,7 +302,11 @@ mrb_vformat(mrb_state *mrb, const char *format, va_list ap)
           mrb_gc_arena_restore(mrb, ai);
           break;
         case 'n':
+#if UINT32_MAX < INT_MAX
+          obj = (mrb_sym)mrb_symbol_value(va_arg(ap, int));
```

Comment:
I thin the line here should be:
``` c
obj = mrb_symbol_value((mrb_sym)va_arg(ap, int));
```
Right?

Reply:
You are right, thank you! I will fix it.